docs/resources.md

```diff
@@ -618,6 +619,213 @@ the container image
 [gcr.io/cloud-builders//gcs-fetcher](https://github.com/GoogleCloudPlatform/cloud-builders/tree/master/gcs-fetcher)
 does not support configuring secrets.
 
+### Cloud Event Resource
+
+The Cloud Event Resource represents a cloud event that is sent to a target `URI`
+upon completion of a `TaskRun`. The cloud event sends Tekton specific events;
+the body of the event includes the entire TaskRun spec plus status; the
+types of events defined for now are:
+
+- dev.tekton.event.task.unknown
+- dev.tekton.event.task.successful
+- dev.tekton.event.task.failed
+
+Cloud event resources are useful to notify a third party upon the completion and
+status of a `TaskRun`. In combinations with the Tekton triggers project they can
```

Comment:
maybe link to https://github.com/tektoncd/triggers ?

Reply:
done